web/client/translations/data.pt-PT.json

```diff
@@ -101,6 +101,11 @@
                 "right": "Right",
                 "bottom": "Bottom",
                 "top": "Top"
+            },
+            "legendOptions": {
```

Comment:
Mandatory translations are :
en, it, es, fr, de. You can avoid to update the other files.